eslint.config.mjs

```diff
@@ -0,0 +1,184 @@
+import noOnlyTests from "eslint-plugin-no-only-tests";
+import globals from "globals";
+import tsParser from "@typescript-eslint/parser";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all
+});
+
+export default [
+  {
+    ignores: [
+      "src/core/cover-service-api/model/",
+      "src/core/cover-service-api/cover-service.ts",
+      "src/core/dpl-cms/model",
+      "src/core/dpl-cms/dpl-cms.ts",
+      "src/core/fbs/fbs.ts",
+      "src/core/publizon/publizon.ts",
+      "postcss.config.js"
+    ]
+  },
+  ...compat.extends(
+    "airbnb",
+    "airbnb-typescript",
+    "airbnb/hooks",
+    "prettier",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+    "plugin:cypress/recommended"
+  ),
+  {
+    plugins: {
+      "no-only-tests": noOnlyTests
+    },
+
+    languageOptions: {
+      globals: {
+        ...globals.browser
+      },
+
+      parser: tsParser,
+      ecmaVersion: 5,
+      sourceType: "module",
+
+      parserOptions: {
+        allowImportExportEverywhere: false,
+
+        ecmaFeatures: {
+          jsx: true,
+          globalReturn: false
+        },
+
+        project: ["./tsconfig.json", "cypress/tsconfig.json"]
+      }
+    },
+
+    settings: {
+      react: {
+        version: "16.11.0"
```

Comment:
Please consider if we need to update this to the version of React that we actually use.